controllers/host/host_controller.go

```diff
@@ -1064,10 +1065,15 @@ func (r *HostReconciler) CompareDisabledAttributes(in *starlingxv1.HostProfileSp
 func (r *HostReconciler) ReconcileHostByState(client *gophercloud.ServiceClient, instance *starlingxv1.Host, current *starlingxv1.HostProfileSpec, profile *starlingxv1.HostProfileSpec, host *v1info.HostInfo) error {
 
 	principal := false
+	requires_vim_strategy := false
```

Comment:
Can you add a comment above regarding this variable? The name is confusing

Reply:
Done.

Reply:
Suggest to rename this variable to "strategy_required"

Reply:
> Suggest to rename this variable to "strategy_required"

Done.